Mathlib/Data/Fin/Basic.lean

```diff
@@ -422,15 +420,8 @@ instance inhabitedFinOneAdd (n : ℕ) : Inhabited (Fin (1 + n)) :=
 theorem default_eq_zero (n : ℕ) [NeZero n] : (default : Fin n) = 0 :=
   rfl
 
-section from_ad_hoc
-
-@[simp] lemma ofNat'_zero [NeZero n] : (Fin.ofNat' n 0) = 0 := rfl
-@[simp] lemma ofNat'_one [NeZero n] : (Fin.ofNat' n 1) = 1 := rfl
-
-end from_ad_hoc
```

Comment:
Should these be deprecated? What replaces them?

Reply:
Testing these with `by simp`, it seems that they have no replacement, and so should not be deleted.

Reply:
Good catch, thanks! Not sure what happened there.